src/Data/Nat/DivMod.agda

```diff
@@ -182,11 +186,11 @@ m<[1+n%d]⇒m≤[n%d] {m} n (suc d-1) = k<1+a[modₕ]n⇒k≤a[modₕ]n 0 m n d-
 ------------------------------------------------------------------------
 -- Properties of _/_
 
-/-congˡ : ∀ {m n o : ℕ} .{{_ : NonZero o}} →
+/-congˡ :  .{{_ : NonZero o}} →
           m ≡ n → m / o ≡ n / o
```

Comment:
```suggestion
/-congˡ : .{{_ : NonZero o}} → m ≡ n → m / o ≡ n / o
```

Reply:
And the subsequent lemma, for uniformity of layout.